flutter_app/lib/drone_information.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:imacs/mavlink_communication.dart';
+
+import 'data_field_widget.dart';
+
+/// Widget to arrange multiple data widgets in tabular form.
+///
+/// This widget  displays all the data fetched from
+/// Mission Planner MAVLink. It automatically sets the data in a two
+/// column layout.
+///
+class DroneInformation extends StatelessWidget {
+  /// @brief Constructs a DroneInformation widget.
+  ///
+  /// @param comm The communication channel (MAVLink) where to get
+  /// the data from
+  ///
+  const DroneInformation({
+    super.key,
+    required this.comm,
+  });
+
+  final MavlinkCommunication comm;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        border: Border.all(
+          color: Colors.black,
+        ),
+      ),
+      child: GridView.count(
+        crossAxisCount: 2, /// describes how many items in one row.
```

Comment:
Kinda an oversight on my part, the formatter wants the comments to be on a separate line. Run `dart format .` in the terminal to fix the build issue